EDDiscovery/3DMap/FormMap.cs

```diff
@@ -501,62 +667,63 @@ private void toolStripButtonGrud_Click(object sender, EventArgs e)
         /// </summary>
         /// <param name="sender"></param>
         /// <param name="e"></param>
-        private void glControl1_Load(object sender, EventArgs e)
+        private void glControl_Load(object sender, EventArgs e)
         {
-            _loaded = true;
 
-            GL.ClearColor((Color)System.Drawing.ColorTranslator.FromHtml("#0D0D10")); 
+            GL.ClearColor((Color)System.Drawing.ColorTranslator.FromHtml("#0D0D10"));
 
             SetupViewport();
-        }
 
-        /// <summary>
-        /// Paint The control.
-        /// </summary>
-        /// <param name="sender"></param>
-        /// <param name="e"></param>
-        private void glControl1_Paint(object sender, PaintEventArgs e)
-        {
-            if (!_loaded) // Play nice
-                return;
+            _loaded = true;
 
-            GL.Clear(ClearBufferMask.ColorBufferBit | ClearBufferMask.DepthBufferBit);
+            Application.Idle += Application_Idle;
```

Comment:
Added this event so we could make more of a "GameLoop" rather than just having the GLControl repaint whenever it feels the canvas has changed.